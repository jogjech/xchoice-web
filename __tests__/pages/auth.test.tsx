import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import AuthPage from "../../pages/auth";

jest.mock("next/router", () => ({
  useRouter: () => {
    return {
      route: "/",
      pathname: "",
      query: { returnTo: "/dashboard" },
    };
  },
}));

describe("Auth page", () => {
  let wrapper: ShallowWrapper;

  beforeEach(() => {
    wrapper = shallow(<AuthPage></AuthPage>);
  });

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
