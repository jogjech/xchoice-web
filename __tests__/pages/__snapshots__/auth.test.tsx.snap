// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth page renders correctly 1`] = `
<div
  className="spinContainer"
>
  <Spin
    size="default"
    spinning={true}
    tip="Redirecting..."
    wrapperClassName=""
  />
</div>
`;
